src/test_verifier/main.cpp

```diff
@@ -51,8 +50,9 @@ int main(int argc, char** argv) {
     int skipped = 0;
     while (true) {
         TestCase test_case;
-        if (!fread(&test_case, sizeof(test_case), 1, file))
+        if (std::fread(&test_case, sizeof(test_case), 1, file) != sizeof(test_case)) {
```

Comment:
The parameters passed here is `object_size = sizeof(test_case)` and `object_count = 1`. The return value of `fread` is the number of object read successfully, which is less than or equal to `object_count`, which can only be 1 (success) or 0 (no read or partial read) in this case. In fact, with this code [CI](https://ci.appveyor.com/project/wwylele/teakra/builds/20986667) can't read any test cases at all:

>1: Test command: C:\projects\teakra\build\src\test_verifier\Release\test_verifier.exe "C:\Users\appveyor\assets/teaklite2_tests_result"
1: Test timeout computed to be: 10000000
1: **0 / 0 passed, 0 skipped**
1/1 Test #1: tests ............................   Passed    0.12 sec